contracts/ERC721AStorage.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+
+import { ERC721AUpgradeable } from "./ERC721AUpgradeable.sol";
+
+library ERC721AStorage {
+
+  struct Layout {
+
```

Comment:
Nit: remove this blank line

Reply:
I'll setup a proper linter for 721A and this repo soon

Reply:
I added a `(npm run lint:sol) || true` line to the `transpile-merge.sh`.